docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,127 @@
+  \f$P_N\f$ scheme and converges at order \f$\mathcal{O}\left(\Delta
+  x^{N+1}\right)\f$ where \f$\Delta x\f$ is the 1d size of the element.
+- The unit normal vector to the element is denoted by \f$n_i\f$.
+- The numerical flux boundary correction term which includes the normal vectors
+  is denoted by \f$G(F^{i,+},n_i^+,u^+,F^{i,-},n_i^-,u^-)\f$ where the \f$+\f$
+  and \f$-\f$ denote the values on the outer and inner side of the mortar,
+  respectively. To give a concrete example, for the Rusanov flux we have
+  \f{align*}{
+      G^{\mathrm{Rusanov}}=
+      \frac{1}{2}\left(F^{i,+}n_i^+-F^{i,-}n_i^-\right)
+      -\frac{C}{2}\left(u^+-u^-\right)
+  \f}
+
+# General Relativity and (Magneto)hydrodynamics{#gr_hydro_notation}
+
+We use the following conventions and notation:
+- In general we work in units where \f$G=c=1\f$.
+- Latin letters at the beginning of the alphabet, \f$a,b,c,\ldots\f$ are
+  spacetime indices ranging from 0 to 3.
+- Latin letters \f$i,j,k,\ldots\f$ are spatial indices ranging from 1 to 3.
+
+The notation for general relativity quantities used is:
+- The Einstein summation convention is used where repeated indices are summed
+  over.
+- The spacetime metric is denoted by \f$g_{ab}\f$ with signature
+  \f$(-,+,+,+)\f$, and its determinant by \f$g\f$.
+- The lapse and shift are denoted by \f$\alpha\f$ and \f$\beta^i\f$,
+  respectively.
+- The spatial metric and its determinant are denoted by \f$\gamma_{ij}\f$ and
+  \f$\gamma\f$, respectively.
+- We write the line element as:
+  \f{align*}{
+      ds^2=g_{ab}dx^a dx^b=-\alpha^2 dt^2 +
+      \gamma_{ij}
+      \left(dx^i + \beta^i dt\right)\left(dx^j + \beta^j dt\right).
+  \f}
+- The unit timelike normal to the hypersurface is
+  \f$n^a=(1/\alpha,-\beta^i/\alpha)\f$ and \f$n_a=(-\alpha,0,0,0)\f$.
+- The extrinsic curvature is denoted by \f$K_{ij}\f$ and its trace by
+  \f$K\f$. We use the sign convention
+  \f{align*}{
+      K_{ab}=-\frac{1}{2}\mathcal{L}_n g_{ab}
+  \f}
+- For the generalized harmonic evolution system \f$\Pi_{ab}=-n^c\partial_c
+  g_{ab}\f$ and \f$\Phi_{iab}=\partial_i g_{ab}\f$.
+
+The notation for (magneto)hydrodynamics used is:
+- The fluid pressure, rest mass density, specific internal energy, and specific
+  enthalpy are denoted by \f$p\f$, \f$\rho\f$, \f$\epsilon\f$, and \f$h\f$,
+  respectively.
+- The 4-velocity of the fluid is \f$u^a\f$ and \f$u^a u_a=-1\f$.
+- The spatial velocity of the fluid as measured by an observer at rest in the
+  spatial hypersurfaces (“Eulerian observer”) is
+  \f{align*}{
+      v^i=\frac{1}{\alpha}\left(\frac{u^i}{u^0}+\beta^i\right)
+  \f}
+  with corresponding Lorentz factor
+  \f{align*}{
+      W=-u^a n_a=\alpha u^0=\frac{1}{\sqrt{1-\gamma_{ij}v^i v^j}}
+  \f}
+- The Faraday tensor is denoted by \f$F^{ab}\f$ and its dual is given by
+  \f$^{*}\!F^{ab}=\frac{1}{2}\epsilon^{abcd}F_{cd}\f$ where
+  \f$\epsilon^{abcd}\f$ is the Levi-Civita tensor.
```

Comment:
Levi-Civita symbol

Reply:
No, you're wrong.

Reply:
That‘s a quite unhelpful comment :smile:

If you want to be precise, and don’t want to write _symbol_, then call it a _tensor density_. Only the dedensitized object `\sqrt{g}\epsilon` transforms like a tensor.

Reply:
It's not the symbol, it's the tensor. I don't see how telling you that is unhelpful. The symbol is 0,1,-1, the tensor has a factor of sqrt{-g} in it (see @carmaza's comment). You are asking me to change the equation from something that is correct to something that is incorrect. See, for example, Eq. 5.131 in Baumgarte and Shapiro.

Reply:
Good, in response to @carmaza's comment you added a definition of the dedensitized tensor.

Reply:
Please don't make it sound like something fundamental changed here. It's the same equation that you thought was incorrect.